Swiftfin/Views/PagingLibraryView/PagingLibraryView.swift

```diff
@@ -282,179 +321,187 @@ struct PagingLibraryView<Element: Poster>: View {
                 router.route(to: \.filter, $0)
             }
         }
-        .onChange(of: defaultDisplayType) { newValue in
-            guard !Defaults[.Customization.Library.rememberLayout] else { return }
-
-            if UIDevice.isPhone {
-                layout = Self.phoneLayout(
-                    posterType: defaultPosterType,
-                    viewType: newValue
-                )
-            } else {
-                layout = Self.padLayout(
-                    posterType: defaultPosterType,
-                    viewType: newValue,
-                    listColumnCount: defaultListColumnCount
-                )
-            }
-        }
-        .onChange(of: defaultListColumnCount) { newValue in
-            guard !Defaults[.Customization.Library.rememberLayout] else { return }
-
-            if UIDevice.isPad {
-                layout = Self.padLayout(
-                    posterType: defaultPosterType,
-                    viewType: defaultDisplayType,
-                    listColumnCount: newValue
-                )
-            }
-        }
-        .onChange(of: defaultPosterType) { newValue in
-            guard !Defaults[.Customization.Library.rememberLayout] else { return }
+    }
 
-            if UIDevice.isPhone {
-                if defaultDisplayType == .list {
-                    collectionVGridProxy.layout()
-                } else {
+    // MARK: body
+
+    // TODO: becoming too large for typechecker during development, should break up somehow
+
+    var body: some View {
```

Comment:
I would actually ask that we don't perform this refactoring here. I will take a look at it sometime.

Reply:
I can reset this back to where it was before. This was just my fix because I hit the typechecker errors when I added the extra if in https://github.com/jellyfin/Swiftfin/pull/980#discussion_r1615015997. If I can avoid that extra if the typerchecker doesn't cause issues on build.